Makefile.PL

```diff
@@ -30,7 +30,7 @@ WriteMakefile(
     },
     PREREQ_PM         => {
         'Gtk3'              => 0,
-        'Gtk3::WebKit2'     => 0.01,
+        'Gtk3::WebKit2'     => 0.011,
```

Comment:
Gtk3::WebKit2 is on 0.012

Reply:
Gtk3::Webkit2 0.012 only includes text changes. 0.011 had some needed changes for the release (seemed to cause problems building on cpan). It doesn't need the newer version.